supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -18,17 +18,23 @@
 *Description*: API objects are the resources that can be exposed by an API server
 at an endpoint for interrogation.
 
-Use initial capitalization and camel case for Kubernetes or OpenShift API
-objects and do not mark them up unless referring to a specific field or variable
-name from a spec or manifest file.
+When referring to terms that are API objects, if it is important to make the distinction that the context is in reference to the actual API object (like when defining or editing an object), use the proper PascalCase `Kind` for the object, and mark it up in monospace. For example:
 
-This matches general Kubernetes usage and makes it obvious that a specific
-concept is being referred to. For example:
+* `Pod`
+* `Node`
+* `DaemonSet`
+* `ConfigMap`
 
-- Pod
-- Deployment
-- Operator
-- DaemonSet (and not "daemon set", "daemonset", or "Daemonset")
+Example object-specific usage: _The default amount of CPU that a container can use if not specified in the `Pod` spec._
+
+However, any other general references to the terms should use lower case, and separate multi-word API objects. For example:
```

Comment:
```suggestion
However, use lowercase for any other general references to the terms, and separate multi-word API objects. For example:
```

IBM style suggestions here: remove the 'should', the passive, and change spelling of lowercase

Reply:
Thanks, removed!